__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,19 @@
+const Itinerary = require("../src/Itinerary");
+const Port = require("../src/Port");
+
+describe("Itinerary", () => {
+  it("can be instantiated", () => {
+    const itinerary = new Itinerary();
+
+    expect(new Itinerary()).toBeInstanceOf(Object);
+  });
+
+  it("can have ports", () => {
+    const dover = new Port("Dover");
+    const calais = new Port("Calais");
+
+    const itinerary = new Itinerary([dover, calais]);
+
+    expect(itinerary.ports).toEqual([dover, calais]);
```

Comment:
Should be using stubs/mocks to replace the Port. As these tests should only relate to the functionality of `Itinerary` and shouldn't depend on `Port`. e.g.
```
const dover = jest.fn();
const calais = jest.fn();

const itinerary = new Itinerary([dover, calais]);

expect(itinerary.ports).toEqual([dover, calais]);
```
In this example I assigned the ports to `jest.fn()`, but really you could assign these ports to any value, as the only thing this test actually checks is that passing an argument to Itinerary assigns that argument to `itinerary.ports`

Reply:
I though I did these on the Itinerary.test.js file, perhaps I didn't save it and went back. Thanks for spotting this.